lib/chat_channel_fetcher.rb

```diff
@@ -16,6 +16,43 @@ def self.structured(guardian)
     }
   end
 
+  def self.all_secured_channel_ids(guardian)
```

Comment:
This should have tests I think

Reply:
It is tested in a way via the bookmarkable query tests, but yes I will add dedicated tests for this section because it is important.

Reply:
Just checked and this channel fetcher is pretty badly tested :( or maybe mostly indirectly tested. Will add a todo to myself to add more.

Reply:
Yes fetcher is very low on tests, don't need to go aggressive on this, but at least when we add a new method it should be tested directly and not indirectly via another test.